packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -531,16 +552,19 @@ const MarqueeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 					animating: false
 				});
 			}
+
+			this.demote();
 		}
 
 		/*
-		 * Starts marquee animation with synchronization, if not already animating
+		 * Starts marquee animation with synchronization, if not already animating and a timer is
+		 * not already active to start.
 		 *
 		 * @param {Number} [delay] Milliseconds to wait before animating
 		 * @returns {undefined}
 		 */
 		tryStartingAnimation = (delay) => {
-			if (this.state.animating) return;
+			if (this.state.animating || this.timerState !== TimerState.CLEAR) return;
```

Comment:
This resolves a bug that this change surfaced but always existed. If Marquee is updated (by state or new props) after it was triggered to start (e.g. by render), it will call `tryStartingAnimation` and restart it which restarts the timer. The effect is more pronounced with this change because it calls `setState` 200ms before the timer would fire so `marqueeOn="render"` instances would take almost 10s to start.

I _think_ it logically makes sense that we wouldn't try to start an animation if a timer existed to start or reset in this narrow case. Note that `tryStartAnimation` is _only_ called from componentDidUpdate().

Reply:
I see.  I think the thought process was... if some condition occurred that changed the state of the marquee, we would want to start based on the new state.  The other approach is, we should work off any existing timers and assume that any changes were not intended to reset the marquee.  I think this second approach makes sense.